### DISCUSSION TITLE
2 Annoying issues encountered

### DISCUSSION THREAD

**Discussion by Bart-Smith** [rern/rAudio#315] [Bug Reports]
Hi,

I like the software first of all, easy to install and to configure.

But i do have two issues which are very annoying, and probably the reason to drop this software and move on.
But i am willing to  give it a chance to proof itself.

Issue 1 :  After 10mins or so, or every reboot, the WiFi is turned on while i turned it off in the System Settings.
Issue 2 : After 10 mins or so, or every reboot, I lost the connection to my NFS share.

I do not have any of these issues on Moode or Volumio, Not hardware related.

**Comment by AndyFx12** [comment]
Hi Bart-Smith,

I don't have this issues - so I think it's not a general problem.

If you expect someone's help, you have to give more information.
- RPi1,2,3,4?
- Which version of rAudio1 did you install?
- Which WiFi-Adapter do you use?
- Which NAS / NFS-share do you use?

regards, Andy
